### ISSUE TITLE
Open in VSCode just opens VSCode and not the working directory

### ISSUE THREAD

**Issue by 2Friendly4You** [gitbutlerapp/gitbutler#3489]
Windows 0.11.0

When i click the button to open the project in VSCode only vscode is opened and not my working directory.

this is the command executed in the background by gitbutler:
```
"C:\Users\schueler\AppData\Local\Programs\Microsoft VS Code\Code.exe" "--open-url" "--" "vscode://fileC:\Programmieren\PointerDiagram/?windowId=_blank"
```

but it should look like this:
```
"C:\Users\schueler\AppData\Local\Programs\Microsoft VS Code\Code.exe" --open-url -- "vscode://file/C:/Programmieren/PointerDiagram/?windowId=_blank"
```

**Comment by mtsgrd** [comment]
Thanks for reporting @2Friendly4You! Does pr #3495 look about right to you?

**Comment by 2Friendly4You** [comment]
Fixed another bug in #3498
